src/main/java/shadows/apotheosis/adventure/affix/socket/SocketHelper.java

```diff
@@ -68,5 +79,14 @@ public static void setSockets(ItemStack stack, int sockets) {
 	public static boolean hasEmptySockets(ItemStack stack) {
 		return getGems(stack).stream().map(GemItem::getGem).anyMatch(Objects::isNull);
 	}
+	
+	public static int getEmptySocket(ItemStack stack) {
```

Comment:
This method should be named `getFirstEmptySocket` as that is what it returns.  `getEmptySocket` would imply getting an object.